client/src/components/EditProfile/EditPhoto/useStyles.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  authWrapper: {
+    display: 'flex',
```

Comment:
You can assign most of these styles to the MUI Box component:

`<Box display="flex" aligntItems="flex-start" justifyContent="space-between"...>`

Reply:
Done.